bucket/quarto.json

```
@@ -0,0 +1,17 @@
+{
+    "version": "0.9.580",
+    "description": "Quarto is open-source scientific and technical publishing system built on Pandoc.",
+    "homepage": "https://quarto.org",
+    "license": "GPL-2.0-only",
+    "url": "https://github.com/quarto-dev/quarto-cli/releases/download/v0.9.580/quarto-0.9.580-win.msi",
+    "hash": "0901b0570cf3c29138cb56a250062f4942290ea4f255a09bac8c437aac07d8a5",
+    "bin": [
+        "Quarto\\bin\\quarto.cmd"
+    ],
```

```suggestion
    "extract_dir": "Quarto",
    "bin": "bin\\quarto.cmd",
```

There is noting outside of `Quarto` folder, so we can just use *extract_dir* instead.